plugins/azure/postgresqlserver/storageAutoGrowthEnabled.spec.js

```diff
@@ -0,0 +1,118 @@
+var expect = require('chai').expect;
+var storage = require('./storageAutoGrowthEnabled');
+
+const listPostgres = [
+    {
+        'sku': {
+            'name': 'B_Gen5_1',
+            'tier': 'Basic',
+            'family': 'Gen5',
+            'capacity': 1
+        },
+        'location': 'eastus',
+        'tags': {},
+        'id': '/subscriptions/dce7d0ad-ebf6-437f-a3b0-28fc0d22117e/resourceGroups/Ali-Recource_Group/providers/Microsoft.DBforPostgreSQL/servers/postgresql-ali',
+        'name': 'postgresql-ali',
```

Comment:
You should never use your own account/subscription Ids
```suggestion
        'id': '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.DBforPostgreSQL/servers/server1',
        'name': 'postgresql-ali',
```

Reply:
Noted and done as suggested.